Code/Framework/AzToolsFramework/AzToolsFramework/API/EditorCameraBus.h

```diff
@@ -58,6 +59,24 @@ namespace Camera
          */
         virtual bool GetActiveCameraPosition(AZ::Vector3& /*cameraPos*/) { return false; }
 
+        /**
+         * Gets the transform of the currently active Editor camera.
+         * The Editor can have multiple viewports displayed, though at most only one is active at any point in time.
+         * (Active is not the same as "has focus" - a different editor pane can have focus, but there's still one
+         * active viewport that's updating every frame, and the others are not)
```

Comment:
suggestion: Do you know what the criteria is for being 'active' (I would have implicitly assumed focus, but the comment mentions that is not the case). Is there an explicit way to change the active camera? It's been a while since I've looked at this sorry 😅